<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Sasha" ignored="3" name="testNGgrouping" tests="3" failures="1" timestamp="10 Feb 2019 13:46:13 GMT" time="0.053" errors="0">
  <testcase name="f" time="0.019" classname="TestNNNNNG.NewTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TestNNNNNG.NewTest.f(NewTest.java:23)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f2" time="0.001" classname="TestNNNNNG.NewTest"/>
  <testcase name="f4" time="0.001" classname="TestNNNNNG.NewTest"/>
  <testcase name="verifyHomepageTitle" time="0.0" classname="TestNNNNNG.firsttestngfile">
    <ignored/>
  </testcase> <!-- verifyHomepageTitle -->
  <testcase name="f3" time="0.0" classname="TestNNNNNG.NewTest">
    <ignored/>
  </testcase> <!-- f3 -->
  <testcase name="f1" time="0.0" classname="TestNNNNNG.NewTest">
    <ignored/>
  </testcase> <!-- f1 -->
</testsuite> <!-- testNGgrouping -->
